### PR TITLE
Feature/dropoff machine

### PR DESCRIPTION
**Pull request by JoepEilander**

### CODE REVIEW

File: Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem.png
Comment (TryFox) [inline Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem.png:1]: Add _D suffix

File: Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem.png.meta
Comment (TryFox) [inline Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem.png.meta:1]: Add _D suffix

File: Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem_ConveirBelt.png
Comment (TryFox) [inline Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem_ConveirBelt.png:1]: Add _D suffix

Comment (TryFox) [reply Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem_ConveirBelt.png:1]: Remove _ between DropoffSystem and ConveirBelt

File: Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem_ConveirBelt.png.meta
Comment (TryFox) [inline Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem_ConveirBelt.png.meta:1]: Add _D suffix

Comment (TryFox) [reply Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem_ConveirBelt.png.meta:1]: Remove _ between DropoffSystem and ConveirBelt

File: Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem_ConveirBelt.png
Comment (Joost-Robat) [inline Assets/Art/2D/Textures/Platform/DropOffSystem/T_DropoffSystem_ConveirBelt.png:1]: Grammer mistake; ConveirBelt -> ConveyorBelt

File: Assets/Art/Materials/Platform/DropoffSystem/M_DropoffSystem_ConveirBelt.mat
Comment (Joost-Robat) [inline Assets/Art/Materials/Platform/DropoffSystem/M_DropoffSystem_ConveirBelt.mat:1]: Grammar mistake, see previous comment.

**Review by Joost-Robat — CHANGES_REQUESTED** [review]
Fix grammar mistakes. After that it's ready for merge.

File: Assets/Art/Materials/Platform/DropoffSystem/M_DropoffSystem_ConveirBelt.mat
Comment (TryFox) [inline Assets/Art/Materials/Platform/DropoffSystem/M_DropoffSystem_ConveirBelt.mat:1]: Remove _ between DropoffSystem and ConveirBelt

File: Assets/Art/Materials/Platform/DropoffSystem/M_DropoffSystem_ConveirBelt.mat.meta
Comment (TryFox) [inline Assets/Art/Materials/Platform/DropoffSystem/M_DropoffSystem_ConveirBelt.mat.meta:1]: Remove _ between DropoffSystem and ConveirBelt

**Review by TryFox — CHANGES_REQUESTED** [review]

**Review by TryFox — APPROVED** [review]

**Review by Joost-Robat — APPROVED** [review]
All good and working! Ready for merge.

**Review by PiterGroot — APPROVED** [review]
lgtm!

**Pull request merged**